actions/app.go

```diff
@@ -32,15 +39,26 @@ func App() *buffalo.App {
 		// Automatically redirect to SSL
 		app.Use(forceSSL())
 
+		// Load .env config file
+		envy.Load()
+
 		// Set the request content type to JSON
 		app.Use(middleware.SetContentType("application/json"))
 
 		if ENV == "development" {
 			app.Use(middleware.ParameterLogger)
 		}
 
-		app.GET("/", HomeHandler)
+		// initialize S3 client session
+		S3 = s3.NewSession(os.Getenv("S3_API_KEY"), os.Getenv("S3_API_SECRET"), os.Getenv("S3_REGION"))
+		S3.Bucket = os.Getenv("S3_BUCKET")
+
+		app.GET("/v1/proctor/ping", PingPong)
```

Comment:
This is part of the API group actually -- I only split mine in Tweaser because there was different auth middleware